### PR TITLE
Coding challenges

### PR DESCRIPTION
**Pull request by CoderQudratov**

### CODE REVIEW

File: src/exercise_1.js [hunk 1]
```diff
@@ -1,34 +1,72 @@
...
+  }
+  return text.trim().length === 0;
 }
+// try {
+//   console.log(isStringEmpty(""));
+//   console.log(isStringEmpty())
+
+// } catch (error) {
+//   console.log(error.message);
+// }
```
Comment (jkalandarov) [inline src/exercise_1.js:27]: Please remove unnecessary comments

File: src/exercise_1.js [hunk 2]
```diff
@@ -55,21 +119,76 @@ function createHashTag(text) {
...
+      2,
+      5
+    )} ${phoneNumberString.slice(5, 7)} ${phoneNumberString.slice(7, 9)} `;
+  }
+  return result;
+}
+try {
+  // console.log(formatPhoneNumber(9309478729));
+} catch (err) {
+  console.log(err.message);
```
Comment (jkalandarov) [inline src/exercise_1.js:150]: Please remove these code blocks since they don't have any effect on the implementation.

**Review by jkalandarov — CHANGES_REQUESTED** [review]

File: src/exercise_1.js (hunk 2 above)
Comment (CoderQudratov) [reply src/exercise_1.js]: Thanks for the tip, I changed the code as you said

Comment (CoderQudratov) [reply src/exercise_1.js]: when results will be declared?

Comment (jkalandarov) [reply src/exercise_1.js]: Soon :)

**Review by jkalandarov — COMMENTED** [review]
Thanks for answers

**Pull request closed without merging**